server/scripts/standalone/service-standalone.sh

```diff
@@ -39,11 +33,22 @@ export XCODE_13_CLANG_VERSION=13.0.0
 export SWIFT_5_5_VERSION=5.5
 export IOS_VERSION_MIN=9.0
 export MACOS_VERSION_MIN=10.7
+export LIB_TAPI_1_6_PATH=/usr/local/tapi1.6/lib
```

Comment:
Do we really use it somewhere? I thought it's something we use like alternative to the official (xcode) tool chain on linux?

Reply:
No, we don't use it per se, but there are some old sdks (build.yml) that still reference it.